n2y/rich_text.py

```diff
@@ -134,7 +135,7 @@ def to_pandoc(self):
 class TextRichText(RichText):
     @classmethod
     def from_plain_text(klass, client, string, block=None):
-        notion_data = mock_notion_rich_text(string)
+        notion_data = mock_rich_text(string)
```

Comment:
Surprised our tests didn't catch this. Guess flake8 did so all is well. Nice!

Reply:
I'm not sure where `from_plain_text` is used. I think it may only be in other plugins.